Lib/test/clinic.test.c

```diff
@@ -5045,39 +5007,6 @@ static int
 Test___init___impl(TestObj *self, PyObject *args)
 /*[clinic end generated code: output=f172425cec373cd6 input=4b8388c4e6baab6f]*/
 
-PyDoc_STRVAR(Test___init____doc__,
```

Comment:
@erlend-aasland You added this back in ec45c513d389510930a62631a21a1dbb3f3aabb7 but I don't know why there is a duplicated entry, one generated by clinic and one that is not. Note that the one generated by clinic has been changed since then while the non-clinic one hasn't change (and I think the docstring is now outdated for the clinic one).